src/MultiplayerMod.AttributeProcessor.MSBuild.Task/ProcessAttributes.cs

```diff
@@ -21,7 +21,9 @@ public class ProcessAttributes : Microsoft.Build.Utilities.Task {
             return ReturnError($"Assembly \"{nameof(AssemblyPath)}\" doesn't exist");
 
         var configuration = Configure(AssemblyPath);
-        var module = ModuleDefMD.Load(configuration.OriginalAssemblyPath);
+        FileStream OriginalAssemblyFile = new FileStream(configuration.OriginalAssemblyPath, FileMode.Open);
+        var module = ModuleDefMD.Load(OriginalAssemblyFile);
```

Comment:
Could you please check, I think it's a good idea to dispose the module itself (`using var module`) and it should dispose the underlying stream as well🤔

Reply:
Good idea! Indeed disposing of the module was enough by itself to fix the issue :)